src/linear_algebra/exterior_algebra.lean

```diff
@@ -0,0 +1,152 @@
+/-
+Copyright (c) 2020 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhangir Azerbayev, Adam Topaz, Eric Wieser.
+-/
+
+import algebra.ring_quot
+import .tensor_algebra
+import group_theory.perm.sign
+
+/-!
+# Exterior Algebras
+
+We construct the exterior algebra of a semimodule `M` over a commutative semiring `R`.
+
+## Notation
+
+The exterior algebra of the `R`-semimodule `M` is denoted as `exterior_algebra R M`.
+It is endowed with the structure of an `R`-algebra. Some results only hold at the level of
+generality of an `S`-module `M` and the space `exterior_algebra S N`.
```

Comment:
I don't understand the meaning of the last sentence.

Reply:
Removed, it was part of the other branch that I decided to leave out for now